packages/schemas/src/player/gamemodes/murdermystery/index.ts

```diff
@@ -31,28 +40,19 @@ export class MurderMystery {
   public lootChests: number;
 
   @Field()
-  public murdererWins: number;
+  public overall: MurderMysteryModeMinusAssassins;
```

Comment:
This class name is very confusing. Makes it seem like all the values are missing assassins stats